bin/waagent2.0

```diff
@@ -3457,9 +3457,21 @@ class RdmaHandler(object):
         """
         Write config string to /dev/hvnd_rdma
         """
-        Log("Set /dev/hvnd_rdma")
-        self.wait_rdma_dev()
-        self.write_rdma_dev_conf()
+        skip_rdma_device = False
```

Comment:
This change is mainly to support CentOS 6.x, 7.x and SUSE. I'm testing on CentOS 6.7 that's why I have to change this file. Is 2.1.x supposed to work on CentOS 6.7?